samples/dapr/ui/Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine AS build
+FROM mcr.microsoft.com/dotnet/sdk:8.0-alpine AS build
 WORKDIR /src
 
 COPY *.csproj .
-RUN dotnet restore -r linux-musl-x64
```

Comment:
Removed the `linux-musl-x64`. I am still learning about Runtime Identifiers and Framework-Dependent Publishing: https://learn.microsoft.com/en-us/dotnet/core/deploying/.

Reply:
Is this architecture what was causing the issue?